### ISSUE TITLE
Cannot read property 'getView' of undefined

### ISSUE THREAD

**Issue by alice-em** [Glavin001/atom-beautify#979]
### Description

I get an error with the title and description listed below, on an html file.  There might be a possible missing closing tag somewhere, which was why I wanted to run beautify to help find out where.

OSX 10.11.15 w/ Atom 1.9.0-dev-38ff433

### Expected Results

I expected pretty code, perhaps even beautiful code (well, at least on how it is visually structured).

```
TypeError: Cannot read property 'getView' of undefined
    at getScrollTop (/Users/alice_em/.atom/packages/atom-beautify/src/beautify.coffee:31:29)
    at beautifyCompleted (/Users/alice_em/.atom/packages/atom-beautify/src/beautify.coffee:108:27)
    at tryCatcher (/Users/alice_em/.atom/packages/atom-beautify/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/Users/alice_em/.atom/packages/atom-beautify/node_modules/bluebird/js/release/promise.js:502:31)
    at Promise._settlePromise (/Users/alice_em/.atom/packages/atom-beautify/node_modules/bluebird/js/release/promise.js:559:18)
    at Promise._settlePromise0 (/Users/alice_em/.atom/packages/atom-beautify/node_modules/bluebird/js/release/promise.js:604:10)
    at Promise._settlePromises (/Users/alice_em/.atom/packages/atom-beautify/node_modules/bluebird/js/release/promise.js:683:18)
    at Async._drainQueue (/Users/alice_em/.atom/packages/atom-beautify/node_modules/bluebird/js/release/async.js:138:16)
    at Async._drainQueues (/Users/alice_em/.atom/packages/atom-beautify/node_modules/bluebird/js/release/async.js:148:10)
    at Async.drainQueues (/Users/alice_em/.atom/packages/atom-beautify/node_modules/bluebird/js/release/async.js:17:14)
```

### Steps to Reproduce

1. Add code to Atom editor
2. Run command `Atom Beautify: Beautify Editor`
3. This beautified code does not look right!

### Debug

Here is a link to the `debug.md` Gist: ( https://gist.github.com/alice-em/b27b1e4b59c969e935e0d1c01fae370d ) (it ain't pretty)

#### How to create `debug.md` Gist

Note that this will include a copy of your code.
If your code is private, please create a different sample of code that reproduces the problem.

1. In the Atom [command-palette](https://atom.io/packages/command-palette), 
search for and run the command `Atom Beautify: Help Debug Editor`. 

The debugging results will be copied to your clipboard.  
2. Create a new Gist at https://gist.github.com/  
3. Create a file in your new Gist called `debug.md`.  
4. Paste your debugging results from Atom beautify into `debug.md` file in your Gist.  
5. Add a link to your Gist in your new Issue.

https://gist.github.com/alice-em/b27b1e4b59c969e935e0d1c01fae370d

# Checklist

- [x] I have tried uninstalling and reinstalling Atom Beautify to ensure it installed properly
- [x] I have reloaded (or restarted) Atom to ensure it is not a caching issue
- [x] Searched for existing Atom Beautify Issues at https://github.com/Glavin001/atom-beautify/issues 
so I know this is not a duplicate issue
- [x] Generated debugging information and added link for `debug.md` Gist to this issue

**Comment by Glavin001** [comment]
> Atom 1.9.0-dev-38ff433

It may be an unsupported Atom API with your version of Atom. I am on v1.7.3.
Unfortunately, I don't have time to investigate. If you do find a solution feel free to submit a Pull Request. Thanks!

**Comment by alice-em** [comment]
I think it might be tied to 1.9.0-dev.  I'm on my Ubuntu machine testing between 1.7.3 and 1.9.0-dev-896abe3, and it works with 1.7.3, but fails with 1.9.0-dev.

**Comment by nfantone** [comment]
Same here on 1.9.0-dev-720ad9c

**Comment by Glavin001** [comment]
https://github.com/Glavin001/atom-beautify/pull/987 has been merged. Awaiting a few other Pull requests to be merged and then will publish a release!

**Comment by FallingSnow** [comment]
I am also experiencing the error. I am running the latest master branch of the atom git repo, version 1.10.0-dev-f6e87b1 on Arch Linux. I am trying to beautify a JavaScript file but this error occurs and the file is not modified at all. Hopefully this fixes the problem when v0.30.0 is released.

```javascript
TypeError: Cannot read property 'getView' of undefined
    at getScrollTop (/home/ayrton/.atom/packages/atom-beautify/src/beautify.coffee:31:29)
    at beautifyCompleted (/home/ayrton/.atom/packages/atom-beautify/src/beautify.coffee:108:27)
    at tryCatcher (/home/ayrton/.atom/packages/atom-beautify/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/home/ayrton/.atom/packages/atom-beautify/node_modules/bluebird/js/release/promise.js:502:31)
    at Promise._settlePromise (/home/ayrton/.atom/packages/atom-beautify/node_modules/bluebird/js/release/promise.js:559:18)
    at Promise._settlePromise0 (/home/ayrton/.atom/packages/atom-beautify/node_modules/bluebird/js/release/promise.js:604:10)
    at Promise._settlePromises (/home/ayrton/.atom/packages/atom-beautify/node_modules/bluebird/js/release/promise.js:683:18)
    at Async._drainQueue (/home/ayrton/.atom/packages/atom-beautify/node_modules/bluebird/js/release/async.js:138:16)
    at Async._drainQueues (/home/ayrton/.atom/packages/atom-beautify/node_modules/bluebird/js/release/async.js:148:10)
    at Async.drainQueues (/home/ayrton/.atom/packages/atom-beautify/node_modules/bluebird/js/release/async.js:17:14)
```

**Comment by Glavin001** [comment]
Published to v0.29.8
